color_helper.py

```diff
@@ -1001,15 +1001,30 @@ def is_enabled(self, mode, palette_name=None, color=None, auto=False):
         )
 
 
-class ColorHelperFileIndexCommand(sublime_plugin.TextCommand):
-    """Color Helper file index command."""
+class ColorHelperFileIndexCommand(sublime_plugin.WindowCommand):
```

Comment:
Curious why this is changed from a text command to a window command?

Reply:
`TextCommand` are made for modifying the content of a view.
This command doesn't need to modify the content of the view so it doesn't need to be a `TextCommand`.

Reply:
While this is true, the command also directly operates on a view and relying on window.active_view prevents someone from using run_command on an inactive view directly.

Reply:
Yes, this is generally how I approach my command type selection. I use Text if it is view specific and only use window if it is not view specific, or there is some advantage using Window over Text. Same goes for Window vs Application.

Reply:
ok changed back to TextCommand